main.css

```diff
@@ -0,0 +1,65 @@
+body {
```

Comment:
This is basically saying, hey `<body>`, do what you would already be doing anyway.  It's not breaking anything, but it's not doing anything either.